File: src/frontend/src/library/teamAggregate/commands/AddTeammateToTeamCommand.ts
export class AddTeammateToTeamCommand {
  public teamId: string;
  public name: string;
  public dateOfBirth: string | null;

  constructor(teamId: string, name: string, dateOfBirth: string) {
    this.teamId = teamId;
    this.name = name;
    this.dateOfBirth = dateOfBirth;
  }
}

File: src/backend/Application/TeamAggregate/Commands/AddTeammateToTeam/AddTeammateToTeamCommand.cs
using InternetSafetyPlan.Application.Base;

namespace InternetSafetyPlan.Application.TeamAggregate.Commands;

public record AddTeammateToTeamCommand(Guid TeamId, string TeammateName, DateTime TeammateDateOfBirth, Guid? UserId = null) : ICommand<Guid>;

